src/lm.jl

```diff
@@ -243,3 +243,28 @@ function confint(obj::LinearModel; level::Real=0.95)
     hcat(coef(obj),coef(obj)) + stderror(obj) *
     quantile(TDist(dof_residual(obj)), (1. - level)/2.) * [1. -1.]
 end
+
+"""
+    cooksdistance(obj::LinearModel)
+
+Compute [Cook's distance](https://en.wikipedia.org/wiki/Cook%27s_distance)
+for each observation in linear model `obj`, giving an estimate of the influence
+of each data point.
+Currently only implemented for LinearModel models without weights.
+"""
+function StatsBase.cooksdistance(obj::LinearModel)
+    u = residuals(obj)
+    mse = dispersion(obj,true)
+    k = dof(obj)-1
+    d_res = dof_residual(obj)
+    X, XtX = crossmodelmatrix(obj)
```

Comment:
oops, I messed this up in my last suggestion, should be:
```suggestion
    X = modelmatrix(obj)
    XtX = crossmodelmatrix(obj)
```